extensions/kotlin/deployment/src/main/java/io/quarkus/kotlin/deployment/KotlinCompilationProvider.java

```diff
@@ -71,8 +71,8 @@ public boolean hasErrors() {
         public void report(CompilerMessageSeverity severity, String s, CompilerMessageLocation location) {
             if (severity.isError()) {
                 if ((location != null) && (location.getLineContent() != null)) {
-                    errors.add(String.format("%s%n%s:%d:%d", location.getLineContent(), location.getPath(), location.getLine(),
-                            location.getColumn()));
+                    errors.add(String.format("%s%n%s:%d:%d%nReason:%s", location.getLineContent(), location.getPath(), location.getLine(),
```

Comment:
Minor remark but should we maybe have a space avec the colon?

Reply:
`avec` -> `does not compute for an english speaker` :P

Kidding aside, yeah that makes sense

Reply:
Fixed

Reply:
Oh my, looks like it's Friday already :)